src/app/my-messages/community-kick-out-message-card/community-kick-out-message-card.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MessagesService } from '../messages.service';
+import { MatDialog } from '@angular/material';
+import { MemberKickedOutNotification } from './member-kicked-out-notification';
+import { DeletableNotificationComponent } from '../deletable-notification/deletable-notification.component';
+
+@Component({
+  selector: 'app-community-kick-out-message-card',
+  templateUrl: './community-kick-out-message-card.component.html',
+  styleUrls: ['./community-kick-out-message-card.component.scss', '../my-messages.component.scss']
```

Comment:
We could use `@import` directive to import `my-messages.component.scss` straight into `community-kick-out-message-card.component.scss`.